app/retail/emails.py

```diff
@@ -443,7 +443,7 @@ def render_admin_contact_funder(bounty, text, from_user):
     return response_html, response_txt
 
 
-def render_funder_stale(github_username, days=30, time_as_str='about a month'):
+def render_funder_stale(github_username, days=60, time_as_str='a couple months'):
```

Comment:
while i think this is the right code change -- @frankchen07 also mentioned in chat that emails are happening sometimes even more regularly than 30 days, so i'm not sure if this will solve the root problem 🤔 

still worth making the change to 60 days IMO

Reply:
will keep an eye out for that 🙌

Reply:
I think it's more that funders who are funding issues are still getting the emails - they usually respond with "I think something's wrong with your system, I've funded a couple issues." 

Perhaps it is just a time elapsed issue, since if I funded something 30 days back and just am waiting on work, I might get sent the funder email.